source/Cosmos.System2/Network/IPv4/Address.cs

```diff
@@ -67,7 +68,17 @@ public bool IsLoopbackAddress()
 
         public bool IsBroadcastAddress()
         {
-            if (address[0] == 255)
+            if ((address[0] == 255) && (address[1] == 255) && (address[2] == 255) && (address[3] == 255))
```

Comment:
Nit: can be written as:
```cs
return address[0] == 0xFF
    && address[1] == 0xFF
    && address[2] == 0xFF
    && address[3] == 0xFF;
```
Or expression body:
```cs
public bool IsBroadcastAddress() =>
    address[0] == 0xFF
    && address[1] == 0xFF
    && address[2] == 0xFF
    && address[3] == 0xFF;
```

Reply:
Yes!